.azure/autotest_template.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  action: 'fly'
+
+jobs:
+- job: sitl_${{ parameters.name }}_test
+  displayName: ${{ format('Cygwin SITL {0} test', parameters.name) }}
+  pool:
+    vmImage: 'win1803'
+  condition: eq(variables['Build.Reason'], 'Schedule')
+  steps:
+  - script: choco install cygwin --params "/InstallDir:C:\Cygwin /NoStartMenu /NoAdmin"
+    displayName: 'Install Cygwin'
+  - script: choco install gcc-g++ python2 python2-devel python2-future python2-lxml python2-pip python2-pexpect python2-numpy git gettext --source cygwin
+    displayName: 'Install Cygwin packages'
+  - script: git submodule update --recursive --init modules/mavlink
+    displayName: Initialize MAVLink submodule
+  - script: C:\Cygwin\bin\bash --login -c "cd $(cygpath '%BUILD_SOURCESDIRECTORY%')/modules/mavlink/pymavlink && python setup.py build install --user"
```

Comment:
I am not sure that this step is needed as mavproxy will install pymavlink with pip. 
I don't know if using ---user for pymavlink and not for mavproxy could cause issue like two pymavlink version are installed

Reply:
We need it because we are testing latest ArduPilot and it might be using a more recent version of pymavlink (or even just new messages).

It's quite unusual, but currently MAVProxy requires a more recent version than ArduPilot, so when MAVProxy is installed latter, pymavlink is updated too.